cmd/table-manager/main.go

```diff
@@ -23,11 +24,12 @@ func main() {
 			},
 		}
 
-		storageConfig storage.Config
-		schemaConfig  chunk.SchemaConfig
-		logLevel      util.LogLevel
+		ingesterConfig ingester.Config
+		storageConfig  storage.Config
+		schemaConfig   chunk.SchemaConfig
+		logLevel       util.LogLevel
 	)
-	util.RegisterFlags(&serverConfig, &storageConfig, &schemaConfig, &logLevel)
+	util.RegisterFlags(&ingesterConfig, &serverConfig, &storageConfig, &schemaConfig, &logLevel)
```

Comment:
The `ingester.Config` adds 24 flags, while we use only one of them in the table manager. That can be confusing. Maybe it's worth it to separate the max chunk age flag out somehow?

Reply:
It's a valid point, but as far as I can see we already have that problem in spades - for instance `storageConfig` has all the flags for S3, Cassandra, Bigtable, etc., and table-manager only works on DynamoDB.

I'll raise it as a separate issue.

Reply:
> table-manager only works on DynamoDB.

We still use table manager to create weekly tables for Cassandra and BigTable.  Just the provisioned capacity thats not used.

Reply:
I sit corrected